tests/transforms/test_SimCLRTransform.py

```diff
@@ -0,0 +1,22 @@
+from PIL import Image
+import unittest
+
+from lightly.transforms.simclr_transform import SimCLRTransform, SimCLRViewTransform
+
+
+class TestSimCLRTransform(unittest.TestCase):
+
```

Comment:
Could you also add a test with `to_tensor = False` and passing a tensor instead of an image?